src/Integration.Vsix/Settings/GeneralOptionsDialogPage.cs

```diff
@@ -33,7 +33,7 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     internal class GeneralOptionsDialogPage : UIElementDialogPage
     {
         public const string PageName = "General";
-        private const string WikiUrl = "https://github.com/SonarSource/sonarlint-visualstudio/wiki";
+        private const string WikiUrl = "https://docs.sonarsource.com/sonarlint/visual-studio/team-features/connected-mode/";
```

Comment:
This link might change as I had to ask kevin for a more detailed sub link. It is mentioned in our Options window with the text "Note: per-user settings are ignored for C/C++ for solutions using connected mode. See the wiki link for more information." so if there is nothing more precise from kevin I think linking to the connected-mode page makes most sense.